linux_os/guide/system/accounts/accounts-banners/banner_etc_issue/rule.yml

```diff
@@ -56,7 +64,8 @@ description: |-
     metadata:
       labels:
         machineconfiguration.openshift.io/role: master
-      name: 75-master-etc-issue
+        machineconfiguration.openshift.io/role: worker
```

Comment:
A mc can only be applied for one pool at the same time. It is meaningless to add one more label here.

Reply:
The matchExpressions can apply MC for both custom pool and worker pool.
```
spec:
  machineConfigSelector:
    matchExpressions:
      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,infra]}
```